resources/sdk/webmessagingjava/extensions/extensions/AuthResponse.java

```diff
@@ -0,0 +1,51 @@
+package cloud.genesys.webmessaging.sdk.extensions;
+
+import java.io.Serializable;
+
+public class AuthResponse implements Serializable {
```

Comment:
This class shouldn't be needed; IIRC it's only used for handing the response for auth code and client credential token requests.